tests/adapter_tests/aws/test_aws_chalice.py

```diff
@@ -315,13 +315,72 @@ def events() -> Response:
                 return slack_handler.handle(chalice_app.current_request)
 
             headers = self.build_headers(timestamp, body)
-            client = Client(chalice_app, Config())
+            client = Client(chalice_app)
             response = client.http.post("/slack/events", headers=headers, body=body)
 
             assert response.status_code == 200, f"Failed request: {response.body}"
             assert_auth_test_count(self, 1)
             assert self.mock_received_requests["/chat.postMessage"] == 1
 
+    @mock.patch(
+        "slack_bolt.adapter.aws_lambda.chalice_lazy_listener_runner.boto3",
+        autospec=True,
+    )
+    def test_lazy_listeners_non_cli(self, mock_boto3):
+        with mock.patch.dict(os.environ, {"AWS_CHALICE_CLI_MODE": "false"}):
+            assert os.environ.get("AWS_CHALICE_CLI_MODE") == "false"
+
+            mock_s3 = mock.MagicMock()
```

Comment:
I wanted to make sure if this is for s3 client. Is it a lambda client? (I might be missing something)

Reply:
My bad. It is the lambda client. Will fix.